rl_games/configs/atari/ppo_breakout.yaml

```diff
@@ -39,41 +38,45 @@ params:
       initializer:
         name: orthogonal_initializer
         gain: 1.41421356237
+
   config:
+    name: BreakoutNoFrameskip-v4_ray
+    env_name: atari_gym
+    score_to_win: 900
+    normalize_value: True
+    normalize_input: False
     reward_shaper:
-      min_val: -1
-      max_val: 1
-      scale_value: 1
-        
+      # min_val: -1
+      # max_val: 1
+      scale_value: 1   
     normalize_advantage: True
     gamma: 0.999
     tau: 0.95
-    learning_rate: 5e-4
-    name: breakout_ppo
-    score_to_win: 900
-    grad_norm: 10
+    learning_rate: 8e-4
+    lr_schedule: None
+    kl_threshold: 0.01
+    
+    grad_norm: 1.0
     entropy_coef: 0.01
-    truncate_grads: True
-    env_name: atari_gym
+    truncate_grads: True 
     e_clip: 0.2
     clip_value: False
-    num_actors: 16
-    horizon_length: 256
-    minibatch_size: 512
-    mini_epochs: 3
+    num_actors: 64
+    horizon_length: 128
+    minibatch_size: 1024
+    mini_epochs: 2
     critic_coef: 1
-    lr_schedule:  None #adaptive
-    kl_threshold: 0.008
     #lr_schedule:  linear
     #schedule_entropy: True
-    normalize_value: True
-    normalize_input: False
-    max_epochs: 3000
+    max_epochs: 1500
+
     env_config:
       skip: 4
       name: BreakoutNoFrameskip-v4
-      episode_life: True
+      episodic_life: True
```

Comment:
for ray this parameter is called: episode_life

Reply:
Are you serious?! Why don't they have the same name?